### PR TITLE
Compatibility with TYPO3 7.6

### PR DESCRIPTION
**Pull request by Oktopuce**

### CODE REVIEW

File: Classes/Hook/ContentPostProc.php [hunk 1]
```diff
@@ -121,7 +121,9 @@ public function main(array $params, \TYPO3\CMS\Frontend\Controller\TypoScriptFro
                         $icons
                     )
                 );
-                $view->getRenderingContext()->setLegacyMode(false);
+                // Method getRenderingContext() available since V8
+                if (method_exists($view,'getRenderingContext'))
```
Comment (MattiasNilsson) [inline Classes/Hook/ContentPostProc.php]: Add a space after the "," in method_exists($view, 'getRenderingContext')

Comment (MattiasNilsson) [inline Classes/Hook/ContentPostProc.php]: Add { } to if statement.

**Comment by Oktopuce** [comment]
Hi Mattias, I've made the changes. You're right I didn't respect the Typo3 coding guidelines.

**Comment by MattiasNilsson** [comment]
Thanks for the PR! 👍

**Pull request merged**